module/app/influxdbconnector/src/main/java/org/openbase/bco/app/influxdbconnector/InfluxDbconnectorApp.kt

```diff
@@ -201,9 +207,9 @@ class InfluxDbconnectorApp : AbstractAppController() {
 
         logger.debug("finish heartbeat")
         if (heartbeat != null && !heartbeat!!.isDone) {
```

Comment:
Use `heartbeat?.takeIf { !it.isDone }?.let {` here

This would simplify the following code and remove `!!` checks